notebooks/priority_fields/get_cred.ipynb

```diff
@@ -0,0 +1,199 @@
+        "            [{\"LOWER\": \"sufficiently\"}, {\"LOWER\": \"consistent\"}],\n",
+        "            [{\"LOWER\": \"testified\"}, {\"LOWER\": \"credibly\"}],\n",
+        "            [{\"LOWER\": \"testimony\"}, {\"LOWER\": \"credible\"}],\n",
+        "            [{\"LOWER\": \"testimony\"}, {\"LOWER\": \"consistent\"}],\n",
+        "\n",
+        "            # least specific phrases\n",
+        "            [{\"LOWER\": \"coherent\"}],\n",
+        "            [{\"LOWER\": \"plausible\"}]\n",
+        "          ]\n",
+        "\n",
+        "matcher.add('credibility', pattern)"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "prk6kZuq6jQH"
+      },
+      "source": [
+        "def similar(target_phrases, file):\n",
+        "    \"\"\"GET RID OF PUNCT\"\"\"\n",
+        "    matcher = Matcher(nlp.vocab)\n",
+        "    matcher.add('target_phrases', target_phrases)\n",
+        "    \n",
+        "    return matcher(file, as_spans=True)"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "FwF5VUzEt_jm"
+      },
+      "source": [
+        "def get_credibility(self):  \n",
+        "    \"\"\"\n",
+        "    Returns whether or not the Respondent was identified as credible by their\n",
+        "    assigned judge / court. \n",
+        "    \"\"\" \n",
+        "    pattern = [\n",
+        "                # most important phrasing \n",
+        "                [{\"LOWER\": \"court\"}, {\"LOWER\": \"finds\"}, {\"LOWER\": \"respondent\"}, {\"LOWER\": \"generally\"}, {\"LOWER\": \"credible\"}],\n",
+        "                [{\"LOWER\": \"court\"}, {\"LOWER\": \"finds\"}, {\"LOWER\": \"respondent\"}, {\"LOWER\": \"testimony\"}, {\"LOWER\": \"credible\"}],\n",
+        "                [{\"LOWER\": \"court\"}, {\"LOWER\": \"finds\"}, {\"LOWER\": \"respondent\"}, {\"LOWER\": \"credible\"},\n",
+        "\n",
+        "                # next in omportance\n",
+        "                [{\"LOWER\": \"credible\"}, {\"LOWER\": \"witness\"}],\n",
+        "                [{\"LOWER\": \"generally\"}, {\"LOWER\": \"consistent\"}],\n",
+        "                [{\"LOWER\": \"internally\"}, {\"LOWER\": \"consistent\"}],\n",
+        "                [{\"LOWER\": \"sufficiently\"}, {\"LOWER\": \"consistent\"}],\n",
+        "                [{\"LOWER\": \"testified\"}, {\"LOWER\": \"credibly\"}],\n",
+        "                [{\"LOWER\": \"testimony\"}, {\"LOWER\": \"credible\"}],\n",
+        "                [{\"LOWER\": \"testimony\"}, {\"LOWER\": \"consistent\"}],\n",
+        "\n",
+        "                # least importance \n",
+        "                [{\"LOWER\": \"coherent\"}],\n",
+        "                [{\"LOWER\": \"plausible\"}]\n",
+        "              ]\n",
+        "\n",
```

Comment:
I really like this approach, good job. I doubt you had time to manually scrape the hrfCases, but do you have an idea of how accurate this is?